spread.yaml

```diff
@@ -516,4 +523,41 @@ suites:
             apt remove -y qemu genisoimage sshpass
             snap remove ubuntu-image
 
+    tests/ootb/:
+        summary: Out-of-the-box experience on specific distributions
+        systems: [-ubuntu-core-*]
+        environment:
+            SKIP_SETUP: yes
+        manual: true
+        prepare-each: |
+            case "$SPREAD_SYSTEM" in
+                debian-*|ubuntu-*)
+                    apt-get update
```

Comment:
I wanted to mirror instructions from snapcraft.io and not convolute this with any layers of indirection.